src/components/Modeler.vue

```diff
@@ -139,7 +145,15 @@ import runningInCypressTest from '@/runningInCypressTest';
 import getValidationProperties from '@/targetValidationUtils';
 import MiniPaper from '@/components/MiniPaper';
 
-import { faCompress, faExpand, faMapMarked, faMinus, faPlus, faStepBackward, faStepForward } from '@fortawesome/free-solid-svg-icons';
+import {
+  faCompress,
+  faExpand,
+  faMapMarked,
+  faMinus,
+  faPlus,
+  faStepBackward,
+  faStepForward,
+} from '@fortawesome/free-solid-svg-icons';
```

Comment:
Do we have a preferred code style for this? Both seem valid for eslint and it would be good to have a consistent standard.

Reply:
The related rules I could find are https://eslint.org/docs/rules/sort-imports and https://eslint.org/docs/rules/object-curly-newline. However, there doesn't appear to be a rule to split imports into multi lines when they reach a certain max length (related: https://github.com/airbnb/javascript/issues/1051). So I'm fine letting PHPStorm handle this for now.

Reply:
> The related rules I could find are https://eslint.org/docs/rules/sort-imports and https://eslint.org/docs/rules/object-curly-newline. However, there doesn't appear to be a rule to split imports into multi lines when they reach a certain max length (related: [airbnb/javascript#1051](https://github.com/airbnb/javascript/issues/1051)). So I'm fine letting PHPStorm handle this for now.

To be clear, I don't mind what I do but it should be consistent between the four of us. Can we arrive at some sort of standard? There must be a setting in PHPStorm that's breaking these onto multiple lines...

Reply:
Just to further this, I've just come across a case where having your PHPStorm apply this formatting is causing a merge conflict. I don't think the rest of us have the setting enabled that's causing this formatting: can you look into it @chipit24?

Reply:
I don't remember changing this setting specifically in PHPStorm so I assumed it was the same for all of us; I'll update my settings!